build/common.project.props

```diff
@@ -23,7 +23,7 @@
     <TargetFrameworksExeForSigning Condition=" '$(IsXPlat)' == 'true' ">$(NETCoreTargetFramework);netcoreapp5.0</TargetFrameworksExeForSigning>
     <TargetFrameworksExeForSigning Condition="'$(DotNetBuildFromSource)' == 'true'">$(TargetFrameworksExe);net7.0</TargetFrameworksExeForSigning>
     <TargetFrameworksLibrary>$(NETFXTargetFramework);$(NetStandardVersion)</TargetFrameworksLibrary>
-    <TargetFrameworksLibrary Condition="'$(DotNetBuildFromSource)' == 'true'">$(NETCoreTargetFramework);$(NETFXTargetFramework);$(NetStandardVersion)</TargetFrameworksLibrary>
```

Comment:
I find this change interesting. 

This used to be something that was explicitly added for source build before, but now it's getting removed? 
https://github.com/NuGet/NuGet.Client/commit/b5f0bb7e10e3a46a8053a02aca6afda45fb69bd1

Reply:
I think the issue was that the netfx TFMs were referenced downstream in SDK. We're eliminating the use of the netfx TFMs in Linux source-build, so this should no longer be necessary.

Reply:
Thanks @nkolev92 - yeah, that was probably needed for other repos that consumed `nuget.client` artifacts. We are removing all targeting of .NET FX from source-build now.

Reply:
Failing CI leg `Apex Tests on Windows` is not caused by my change, which is source-build (Linux) only.